mmaction/core/hooks/output.py

```diff
@@ -0,0 +1,52 @@
+import functools
+
+
+class OutputHook:
+
+    def __init__(self, module, outputs=None, as_tensor=False):
+        self.outputs = outputs
+        self.as_tensor = as_tensor
+        self.layer_outputs = {}
+        self.handles = []
+        self.register(module)
+
+    def register(self, module):
+
+        def hook_wrapper(name):
+
+            def hook(model, input, output):
+                if self.as_tensor:
+                    self.layer_outputs[name] = output
+                else:
+                    self.layer_outputs[name] = output.detach().cpu().numpy()
```

Comment:
Is this compatible with models such as Slowfast whose output of `backbone` is a tuple if Tensor?

Reply:
For other type, now it will directly return it.